doc/api/worker_threads.md

```diff
@@ -428,7 +428,8 @@ if (isMainThread) {
     not automatically be piped through to `process.stderr` in the parent.
   * `execArgv` {string[]} List of node CLI options passed to the worker.
     V8 options (such as `--max-old-space-size`) and options that affect the
-    process (such as `--title`) are not supported.
+    process (such as `--title`) are not supported. If set, this will be provided
+    as [`process.execArgv`][] inside the worker.
```

Comment:
```suggestion
    as [`process.execArgv`][] inside the worker. By default, this is set to
    `process.execArgv`.
```

Reply:
That’s not accurate, though. `process.execArgv` may contain options that cannot be used in this argument, but that’s still valid.

Reply:
I updated the suggestion.

Reply:
I added `By default, options will be inherited from the parent thread.`, which should be short and accurate. (Unless your intention is to state something else here?)